### DISCUSSION TITLE
Performer URL Field

### DISCUSSION THREAD

**Discussion by elkorol** [stashapp/stash#3148] [Ideas]
Hi there, I propose making the URL field for performers, more like the Studio aliases. One default existing, but with a button to add extra text entry fields for extra URLs. I think this would be better because some performers have more than one url associated with them.

Thanks

**Comment by QxxxGit** [comment]
This is currently requested #703, which appears to be planned for [v0.19](https://github.com/stashapp/stash/milestone/23).

**Reply by elkorol** [reply]
Ah I see, i should have checked that. Didn't know of checking upcoming milestones for versions. Thanks
